case-lib/lib.sh

```diff
@@ -343,3 +343,19 @@ disable_kernel_check_point()
 {
     KERNEL_CHECKPOINT="disabled"
 }
+
+is_sof_logger_disabled()
```

Comment:
I think `logger_disabled()` is long enough name.

Reply:
@marc-hb ， /etc/sof/manifest.txt has below information:
```
{
    "date" : "2021-05-21 07:35:42 UTC",
    "version" : {
        "kernelBranch" : "topic/sof-dev",
        "kernelCommit" : "dcea8052",
        "sofBranch" : "main",
        "sofCommit" : "72a51a462fb8"
        "firmwaretype" : "zephyr"
    },
    "filepath" : {
        "kernel" : "http://sof-ci-storage.sh.intel.com/release/linux/topic/sof-dev/dcea8052/linux-image-5.13.0-rc1-daily-nocodec-20210520-0_5.13.0-rc1-topic-sof-dev-dcea8052_amd64.deb",
        "firmware" : "http://sof-ci-storage.sh.intel.com/release/sof/main/72a51a462fb8/fw/zephyr/sof-apl.ri",
        "ldc" : "http://sof-ci-storage.sh.intel.com/release/sof/main/72a51a462fb8/fw/zephyr/sof-apl.ldc",
        "lst" : "http://sof-ci-storage.sh.intel.com/release/sof/main/72a51a462fb8/fw/zephyr/sof-apl.lst",
        "topology" : "http://sof-ci-storage.sh.intel.com/release/sof/main/72a51a462fb8/tplg/sof-apl-nocodec.tplg",
        "logger" : "http://sof-ci-storage.sh.intel.com/release/sof/main/72a51a462fb8/tools/sof-logger",
        "ctl" : "http://sof-ci-storage.sh.intel.com/release/sof/main/72a51a462fb8/tools/sof-ctl"
    }
}
```
The firmwaretype is added in the deployment stage. related code is in sof-sh/deploy.sh:
```
# generate manifest file
save_manifest() {
    # save Manifest file to DUT
    cat << END > $MANIFEST_FILE
{
    "date" : "$TODAY",
    "version" : {
        "kernelBranch" : "$KERNEL_BRANCH",
        "kernelCommit" : "$LINUX_SHA",
        "sofBranch" : "$SOF_BRANCH",
        "sofCommit" : "$SOF_SHA"
        "firmwaretype" : "$FW_TYPE"
    },
    "filepath" : {
        "kernel" : "$KERNEL_URL",
        "firmware" : "$FW_URL",
        "ldc" : "$LDC_URL",
        "lst" : "$LST_URL",
        "topology" : "$TPLG_URL",
        "logger" : "$LOGGER_URL",
        "ctl" : "$CTL_URL"
    }
}
END
```
so we will generate the manifest.txt every time when deploy sof environment.

Reply:
@keqiaozhang Please align variable naming style, `fwType` or `firmwareType`

Reply:
Updated.

Reply:
@keqiaozhang To facilitate running sof-test standalone, the manifest file should be documented somewhere within sof-test repository (an example manifest file perhaps).

Reply:
@kv2019i , most of our test scripts have "-s" option, you can set it as 0 to skip the trace collection. for example:
```
./check-playback.sh -d 10 -l 1 -r 1 -s 0
```
if you want to run "run-all-tests.sh" without logger, we can figure out a new solution for it next.

Reply:
>  If you want to run "run-all-tests.sh" without logger, we can figure out a new solution for it next.

I have a hack that does this here: https://github.com/marc-hb/sof-test/commit/run-all-nologger